scipy/linalg/_basic.py

```diff
@@ -1117,7 +1118,7 @@ def solve_circulant(c, b, singular='raise', tol=None,
 
 # matrix inversion
 @_apply_over_batch(('a', 2))
```

Comment:
IIRC this wouldn't be needed anymore?
```suggestion
```

Reply:
Yes, totally. ATM though, this is `inv0`, temporarily kept to simplify perf comparisons.